scripts/results_snapshot.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+CURRENT_DATE=$(date "+%d%m%y")
+FILE_NAME="snapshot_$CURRENT_DATE.tar.gz"
+
+cd ../
+if [ -d "snapshot" ]; then
+  rm -r snapshot
+fi
+mkdir snapshot
+mkdir snapshot/results/
+mkdir snapshot/scripts/
+cp -vr results/ snapshot/results/
+cp -vr scripts/ snapshot/scripts/
+tar -czvf $FILE_NAME snapshot/
+
+rm -r snapshot
```

Comment:
Newline. You know, you can set up your editor so it automatically fixes missing end-of-file newlines...